src/Tribe/Views/V2/Template_Bootstrap.php

```diff
@@ -354,21 +354,28 @@ public function filter_add_body_classes( $classes ) {
 	 * @param string $file      Complete path to include the PHP File
 	 * @param array  $name      Template name
 	 * @param object $template  Instance of the Tribe__Template
-	 * 
+	 *
```

Comment:
:no_entry_sign: Spaces must be used to indent lines; tabs are not allowed

<sub>hash: 3b6b0b3bb54cef0103dcad9a93ce4847</sub>